### PR TITLE
Update README.md to change the example api address

### PR DESCRIPTION
**Pull request by MicroCBer**
It seems like chrome (including edge and other chrome-based browsers) allows access to `ws://localhost` and `ws://127.0.0.1` in a secure context, so it's better to write these two rather than `ws://0.0.0.0`.

(verified by me and @WGzeyu)

image
image

### CODE REVIEW

**Comment by ReadieFur** [comment]
TL;DR: I deliberatly didnt do this becuase external access may be needed.
The reason I never set the server address to localhost/127.0.0.1 is for users who may need to access the data outside of the local machine (like myself). While using a secure connection would be nice, the data sent holds no importance so it isn't so bad if the packets are sniffed, and in reality, who will be sniffing these packets on a network, its unlikley you are playing on a public unsecure network.
Now what I could do is store a mode inside a config file or add a toggle onto the ingame UI to allow or disallow external connections but for the average user, they wouldnt care about this and I also can't quite be bothered to figure out how to add things to the beatsaber UI (iirc theres a library for that but still).

**Comment by ReadieFur** [comment]
Also to add a note, the actual client programs can still connect to this socket securley on the local machine, that is exactly what my overlay is meant to do too.

**Comment by MicroCBer** [comment]
so what i meant is to change the address in readme, not to change the one in Server.cs

**Comment by ReadieFur** [comment]
Well your edit would be wrong anyway, the readme is staing that the data is sent out on that address, you changed it to make it appear like it is sent out on localhost, which it isnt.

**Comment by MicroCBer** [comment]
the reason I want to use it in a secure context is many web apis are only available in secure context. I'm not interested in make it safer, the beatsaber data doesn't need to be that safe :)

**Comment by MicroCBer** [comment]
> Well your edit would be wrong anyway, the readme is staing that the data is sent out on that address, you changed it to make it appear like it is sent out on localhost, which it isnt.

hmm

**Comment by MicroCBer** [comment]
btw the plugin is kinda broken in 1.24

**Comment by ReadieFur** [comment]
Ive not played in months xp. If i get my vr bits setup again soon then ill see if I rembeber to take a look at it again

**Comment by MicroCBer** [comment]
you can use `fpfc` actually

anyway, probably I'll make a pr when i have free time

**Comment by ReadieFur** [comment]
> you can use `fpfc` actually
> 
> anyway, probably I'll make a pr when i have free time

Yeah I'm aware, its just when I dont play it I loose the intrest/motivation to go back to keeping this up to date.
About the pr, if and when you make it ill check it out and well see from there
